src/zookeeper/build/run.sh

```diff
@@ -20,9 +20,19 @@
 sed -i "/^JAVA_OPTS.*/ s:.*:JVMFLAGS=\"${JAVA_OPTS}\":" /usr/share/zookeeper/bin/zkEnv.sh
 
 HOST_NAME=`hostname`
-/usr/local/host-configure.py -c /host-configuration/host-configuration.yaml -f /etc/zookeeper/conf/zoo.cfg -n $HOST_NAME
-cp /myid /var/lib/zoodata/myid
-/usr/local/host-configure.py -c /host-configuration/host-configuration.yaml -f /var/lib/zoodata/myid -n $HOST_NAME
+
+# create /etc/zookeeper/conf/zoo.cfg
+echo "
+tickTime=2000
+dataDir=/var/lib/zoodata
+clientPort=2181
+initLimit=5
+syncLimit=2
+${ZK_SERVERS}
+" > /etc/zookeeper/conf/zoo.cfg
+
+# create /var/lib/zoodata/myid
+echo $HOST_NAME | sha1sum | grep -Eo "[a-f0-9]{6}"  | head -n1 > /var/lib/zoodata/myid
```

Comment:
An example here in comments.

Reply:
It should generate id like below:

> hayua@stcvl-131:~/pai$ echo $HOST_NAME | sha1sum | grep -Eo "[a-f0-9]{6}"  | head -n1
> adc83b

Reply:
Maybe you should write these detail information or knowledge in the document. I think zkid is a very important configuration for zk.

Reply:
Agree, I put some comments.